README.adoc

```diff
@@ -6,6 +6,37 @@ image:https://codecov.io/gh/codeready-toolchain/toolchain-common/branch/master/g
 
 This repo is for controllers, libs, scripts, make files, etc to be shared between host and member operators.
 
+== Setting Up and Connecting Host and Member Operators
+
+In a new terminal, navigate to codeready-toolchain/host-operator directory and execute the following commands
+```
+$ minishift start --profile toolchain
+$ make up-local
+^C
+```
+In the same terminal, navigate to codeready-toolchain/member-operator directory and execute the following commands
+```
+$ make up-local
+^C
+$ make add-member-to-host
+```
+In the same terminal, navigate back to codeready-toolchain/host-operator directory and execute the following commands
+```
+make add-host-to-member
+```
+In the same terminal, navigate back to codeready-toolchain/member-operator directory and execute the following commands
+```
+$ make up-local
```

Comment:
Is there any reason why are we doing `make up-local` on member-operator two times?

Reply:
@dipak-pawar The second time is to get it up and running so we can actually watch the logs now that it is aware of the host. You could alternatively, use two terminals and then not need to start and stop them. Would that be better in the instructions?

Reply:
@dipak-pawar Fixed the typo - good catch :-)

Reply:
> You could alternatively, use two terminals and then not need to start and stop them.

I don't think so. In your instructions, you use two clusters (two profiles) and I'm not sure if there is an easy way you could reach them without switching the profiles/context. If it is possible to do that, then you should describe it in the description. 

> The second time is to get it up and running so we can actually watch the logs now that it is aware of the host.

This is fine for the member, but then if you want to do the same for the host you would need:
1. to stop the process
2. change the profile
3. change the directory
4. run `up-local` for the host. 

Wouldn't be better - instead of starting/stopping the process - to just deploy the operator and let it running inside of the cluster? Then, you could just change the profiles and namespaces

Reply:
@MatousJobanek i have updated the instructions already to avoid stopping the host cluster. But the member does need to be started and stopped in order to run add to host and add to member scripts - as mentioned by @nurali-techie here: https://github.com/codeready-toolchain/toolchain-common/pull/19#issuecomment-512085257

Reply:
the operators don't have to be started in order to create KubeFedCluster the CRs, they can be started later. The Nurali's steps are technically correct, but I wouldn't show setting up clusters by starting/stopping operators without enough explanation

Reply:
@MatousJobanek Done! Agreed - much clearer now.